arches/app/src/components/ControlledListManager/api.ts

```diff
@@ -0,0 +1,126 @@
+import arches from "arches";
```

Comment:
why declare these functions in an API? Shouldn't they live in their call sites?

Reply:
`postListToServer` is being used twice, the others, too early to tell, but maybe/maybe not.  I'm open to moving these around, but I like the idea of centralizing the functions to talk to the server. Keeps the SFCs shorter, and makes it easier to copy/paste boilerplate, ensuring you're following a central set of patterns. One drawback is having to pass `toast` and `$gettext` around, though, so I'm suggestible.